src/Illuminate/Database/composer.json

```diff
@@ -24,6 +24,9 @@
         "illuminate/macroable": "^10.0",
         "illuminate/support": "^10.0"
     },
+    "conflict": {
```

Comment:
I don't think individual components should have different dependencies and conflicts to the main framework package. We should add this conflict to the laravel/framework package too, otherwise this conflict doesn't do anything the event that laravel/framework acts as a replacement for this package.